test/C/Timeout.lf

```diff
@@ -0,0 +1,47 @@
+/*
+ * A test for the timeout functionality in Lingua Franca.
+ * 
+ * @author Soroush Bateni
+ */
+ target C {
+    timeout: 11 msec
+};
+
+import Sender from "DistributedLoopedAction.lf"

```

Comment:
Seems a little odd to import from that file. We should put together a library of commonly used reactors...